templates/home/index.html

```diff
@@ -407,13 +407,13 @@ <h4>IFSC Code :</h4>
                 <div class="contact-details-div">
                      <div class="contact-details contact-details-left">
                         Name <br>
-                        98726 - 829xx <br>
-                        xyxysa@iiitdmj.ac.in
+                        <a style="color: #000;" href="tel:98726 - 829xx" onMouseOver="this.style.color='#ff8c00'"onMouseOut="this.style.color='#000'" > 98726 - 829xx </a><br>
+                        <a style="color: #000;" href="mailto:xyxysa@iiitdmj.ac.in" onMouseOver="this.style.color='#ff8c00'"onMouseOut="this.style.color='#000'" > xyxysa@iiitdmj.ac.in </a>
```

Comment:
Instead of using `onMouseOver` even for replicating hover behaviour, you should write CSS for this and use the `:hover` selector.

You can write the CSS for this page in `style,css` file present in `static/home/css` directory.

Make sure you write the CSS at the proper place in the file (the CSS file is divided into sections according to the sections on the page).